packages/core/content-releases/server/src/services/__tests__/release.test.ts

```diff
@@ -295,4 +445,80 @@ describe('release service', () => {
       });
     });
   });
+
+  describe('deleteAction', () => {
+    it('deletes the action', async () => {
+      const strapiMock = {
+        ...baseStrapiMock,
+        db: {
+          query: jest.fn().mockReturnValue({
+            delete: jest.fn().mockReturnValue({ id: 1, type: 'publish' }),
+          }),
+        },
+      };
+
+      // @ts-expect-error Ignore missing properties
+      const releaseService = createReleaseService({ strapi: strapiMock });
+
+      const release = await releaseService.deleteAction(1, 1);
+
+      expect(release).toEqual({ id: 1, type: 'publish' });
+    });
+
+    it('throws an error if the release does not exist', () => {
```

Comment:
This is kind of confusing since the test should be for a release that _does exist_ but is already published. I think the message should be:

```suggestion
    it('throws an error if the release was already publish', () => {
```

as you have above for the createAction test

Reply:
I change the name to use both, because it's actually the two cases, findOne could return null if the release id is wrong or the release is published

Reply:
I'm not sure I completely understand shouldn't we test cases for each then? 
1. An error is thrown for a release that doesn't exist. 
2.An error is thrown for a release that does exist but already published?

Reply:
Yeah but what happens in these two cases is the same: `update` or `delete` instead of return the updated/deleted entry, returns null so both tests would be exactly the same

Reply:
ok sounds good, I think we will revisit and refactor all the tests anyway when we add the e2e tests next week.